docs/ops/doc/examples/labeling_mesh.rst

```diff
@@ -0,0 +1,244 @@
+            logical operations.
+        
+            :param mask_a:
+        
+                Input mask "A", data to extract.
+        
+            :param mask_b:
+        
+                Input mask "B", region to extract from.
+        
+            :return:
+            
+                Mask with extracted "B" region with "A" data.
+            """
+            # create Img containers
+            tmp = ops.op("create.img").input(mask_a, BitType()).apply()
+            out = ops.op("create.img").input(mask_a, BitType()).apply()
+        
+            # perform logical operations on masks
+            ops.op("logic.or").input(mask_a, mask_b).output(tmp).compute()
+            ops.op("logic.xor").input(tmp, mask_b).output(out).compute()
+            ops.op("copy.img").input(out).output(tmp).compute()
+            ops.op("logic.xor").input(tmp, mask_a).output(out).compute()
+        
+            return out
+        
+        def find_axis_index(image, axis_label):
+            """Find the index of the given axis label.
+        
+            Find the axis index of the given axis label. If no
+            label match is found, return None.
+        
+            :param image:
+        
+                Input Img.
+        
+            :param axis_label:
+        
+                Axis label to find.
+        
+            :return:
+        
+                The index of the given axis label in the image.
+            """
+            for i in range(len(image.dimensionsAsLongArray())):
+                if axis_label == image.axis(i).type().toString():
+                    return i
+                else:
+                    continue
+        
+            return None
+        
+        
+        def gaussian_subtraction(image, sigma):
+            """Perform a Gaussian subtraction on an image.
+        
+            Apply a Gaussian blur and subtract from input image.
+        
+            :param image:
+        
+                Input Img.
+        
+            :param sigma:
+        
+                Sigma value.
+        
+            :return:
+        
+                Gaussian blur subtracted image.
+            """
+            blur = ops.op("filter.gauss").input(image, sigma).apply()
+            out = ops.op("create.img").input(image, FloatType()).apply()
+            ops.op("math.sub").input(image, blur).output(out).compute()
+        
+            return out
+        
+        # extract channels
+        ch_a_img = extract_channel(img, ch_a)
+        ch_b_img = extract_channel(img, ch_b)
+        
+        # customize the following sections below for your own data
+        # clean up channel "A" and create a mask
+        ch_a_img = gaussian_subtraction(ch_a_img, 8.0)
+        ch_a_ths = ops.op("create.img").input(ch_a_img, BitType()).apply()
+        ops.op("threshold.triangle").input(ch_a_img).output(ch_a_ths).compute()
+        ch_a_mask = ops.op("morphology.open").input(ch_a_ths, HyperSphereShape(1), 4).apply()
```

Comment:
I think seeing the neighborhood will make this easier to explain. A `HyperSphereShape` of radius 1 on a 2D plane looks like this:

![sphere_radius_2d](https://github.com/user-attachments/assets/2dfcb19e-ed73-4111-8e6b-a25398ad5e24)

The pattern is essentially a four connected structuring element. You're correct that the neighborhoods are _n_-dimensional. Here's the same `HyperShpereShape` of radius 1 in 3D:

![radius 3d](https://github.com/user-attachments/assets/cc772e94-f1ea-4a9a-9aab-27c786d7c285)

In this case we want to consider the XYZ data per channel, so a `HyperSphereShape`.

We do want to fill holes on the first channel, the puncta channel, because my segmentation strategy successfully extracts the puncta (Gaussian blur subtraction) and they are generally whole. We _ideally_ fill holes on the nuclear channel because the variable signal in the nucleus can produce internal regions that fall below threshold values. In this case, I wanted a whole nucleus to mask over the puncta in the logical operation steps, thus filling holes makes sense here. Ultimately however, its not needed. The threshold masks are good enough for this dataset and the fill holes Op is very slow.